src/parser.rs

```diff
@@ -5842,14 +5875,17 @@ impl<'a> Parser<'a> {
         let (privileges, objects) = self.parse_grant_revoke_privileges_objects()?;
 
         self.expect_keyword(Keyword::TO)?;
-        let grantees = self.parse_comma_separated(Parser::parse_identifier)?;
+        let grantees =
+            self.parse_comma_separated(|p| p.parse_identifier().map(WithSpan::unwrap))?;
 
         let with_grant_option =
             self.parse_keywords(&[Keyword::WITH, Keyword::GRANT, Keyword::OPTION]);
 
         let granted_by = self
             .parse_keywords(&[Keyword::GRANTED, Keyword::BY])
-            .then(|| self.parse_identifier().unwrap());
```

Comment:
I've fixed this previous possible `panic!` in the parser for these `GRANT` statement.